deploy/olm/csv-rhel/storageos.clusterserviceversion.yaml

```diff
@@ -346,9 +352,17 @@ spec:
                       fieldPath: metadata.annotations['olm.targetNamespaces']
                 - name: OPERATOR_NAME
                   value: cluster-operator
+                - name: POD_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: ENABLE_SCHEDULER
+                  value: "true"
                 ports:
-                - containerPort: 8080
+                - containerPort: 60000
```

Comment:
I think this is not right. Tried to make it same as in the operator deployment file. As per https://www.iana.org/form/ports-services this will be out of range `1024 - 49151`. Will set this back to 8080 for now. But it's unused at the moment. It was part of the deployment config generated by the operator-sdk initially.